src/main/scala/it/pagopa/interop/authorizationmanagement/model/persistence/commands.scala

```diff
@@ -76,20 +76,22 @@ final case class UpdateEServiceState(
 ) extends Command
 
 final case class UpdateAgreementState(
-  eServiceId: String,
+  eserviceId: String,
   consumerId: String,
   agreementId: UUID,
   state: PersistentClientComponentState,
   replyTo: ActorRef[StatusReply[Unit]]
 ) extends Command
 
 final case class UpdateAgreementAndEServiceState(
-  eServiceId: String,
+  eserviceId: String,
   descriptorId: UUID,
   consumerId: String,
   agreementId: UUID,
   agreementState: PersistentClientComponentState,
-  descriptorState: PersistentClientComponentState,
+  eserviceState: PersistentClientComponentState,
```

Comment:
```suggestion
  descriptorState: PersistentClientComponentState,
```

Reply:
@galales are you sure about this? This state will affect eService chain state.